sysdig/resource_sysdig_secure_team.go

```diff
@@ -0,0 +1,167 @@
+package sysdig
+
+import (
+	"github.com/draios/terraform-provider-sysdig/sysdig/secure"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"strconv"
+	"time"
+)
+
+func resourceSysdigSecureTeam() *schema.Resource {
+	timeout := 30 * time.Second
+
+	return &schema.Resource{
+		Create: resourceSysdigTeamCreate,
+		Update: resourceSysdigTeamUpdate,
+		Read:   resourceSysdigTeamRead,
+		Delete: resourceSysdigTeamDelete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(timeout),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"theme": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "#73A1F7",
```

Comment:
#73A1F7 is this color
![image](https://user-images.githubusercontent.com/1150301/76073844-a3476e00-5fdd-11ea-8e72-12f4ed9bdbca.png)

Reply:
Perfect!